vscode-web/run.sh

```diff
@@ -1,21 +1,49 @@
-#!/usr/bin/env sh
+#!/usr/bin/env bash
 
 BOLD='\033[0;1m'
+EXTENSIONS=("${EXTENSIONS}")
 
-# Create install directory if it doesn't exist
-mkdir -p ${INSTALL_DIR}
+# Create install prefix
+mkdir -p ${INSTALL_PREFIX}
 
-printf "$${BOLD}Installing vscode-cli!\n"
+printf "$${BOLD}Installing Microsoft Visual Studio Code Server!\n"
 
-# Download and extract code-cli tarball
-output=$(curl -Lk 'https://code.visualstudio.com/sha/download?build=stable&os=cli-alpine-x64' --output vscode_cli.tar.gz && tar -xf vscode_cli.tar.gz -C ${INSTALL_DIR} && rm vscode_cli.tar.gz)
+# Download and extract vscode-server
+ARCH=$(uname -m)
+case "$ARCH" in
+  x86_64) ARCH="x64" ;;
+  aarch64) ARCH="arm64" ;;
+  *)
+    echo "Unsupported architecture"
+    exit 1
+    ;;
+esac
+
+HASH=$(curl https://update.code.visualstudio.com/api/commits/stable/server-linux-$ARCH-web | cut -d '"' -f 2)
+outut=$(curl -sL https://vscode.download.prss.microsoft.com/dbazure/download/stable/$HASH/vscode-server-linux-$ARCH-web.tar.gz | tar -xz -C ${INSTALL_PREFIX} --strip-components 1 && rm -rf vscode-server-linux-$ARCH-web.tar.gz)
 
 if [ $? -ne 0 ]; then
-  echo "Failed to install vscode-cli: $output"
+  echo "Failed to install vscode-server: $output"
   exit 1
 fi
-printf "🥳 vscode-cli has been installed.\n\n"
+echo "🥳 vscode-server has been installed in ${INSTALL_PREFIX}\n"
+
+VSCODE_SERVER="${INSTALL_PREFIX}/bin/code-server"
+
+# Install each extension...
+IFS=',' read -r -a EXTENSIONLIST <<< "$${EXTENSIONS}"
+for extension in "$${EXTENSIONLIST[@]}"; do
+  if [ -z "$extension" ]; then
+    continue
+  fi
+  printf "🧩 Installing extension $${CODE}$extension$${RESET}...\n"
+  output=$($VSCODE_SERVER --install-extension "$extension" --force)
+  if [ $? -ne 0 ]; then
+    echo "Failed to install extension: $extension: $output"
+    exit 1
+  fi
```

Comment:
Given:

```sh
EXTENSIONS="foo,bar,baz"
```

I would suggest the following posix way, so we don't need to rely on bash/arrays.

```suggestion
extension=
while [ "$${extension}" != "$${EXTENSIONS}" ]; do
  extension=$${EXTENSIONS%%,*}
  EXTENSIONS=$${EXTENSIONS#*,}
  if [ -z "$${extension}" ]; then
    continue
  fi
  printf "🧩 Installing extension $${CODE}$extension$${RESET}...\n"
  output=$("$${VSCODE_SERVER}" --install-extension "$${extension}" --force)
  if [ $? -ne 0 ]; then
    echo "Failed to install extension: $${extension}: $${output}"
    exit 1
  fi
```

Reply:
This section is borrowed from [`code-server`](https://github.com/coder/modules/blob/b299f98161a02fe335c7e06ff4c93ce8fc24137a/code-server/run.sh#L28), and I did not try to improve it as code-server is our top downloaded module and well tested. But I agree and we can update.

Reply:
@mafredri, any second thoughts? Otherwise I should also update code-server module.